src/DummyTest.java

```diff
@@ -143,6 +143,74 @@ public void exceptionTestingHealth() {
         // Assert
         assertThrows(IllegalArgumentException.class, executable, "Exception handling failed");
     }
+
+    @Test
+    @DisplayName("Test for battle function")
+    void testForBattle1() {
+        // Arrange
+        Entity entity1 = new Entity(100, 50, 35, 2, "Bob", "Bob story");
+        Entity entity2 = new Entity(90, 40, 30, 2, "Mike", "Mike story");
+        var printer = System.out;
+        Battle battle = new Battle(printer);
```

Comment:
Ha idáig `Entity` volt ebben az osztályban tesztelve, akkor most hirtelen ne legyen itt a `Battle` is tesztelve, valamint ha az `Entity` van itt tesztelve, akkor az tükröződjön is az osztály nevén. Tudtommal nincsen `Dummy` nevű osztályunk, amit tesztelni kéne.

Továbbá, a printer az legyen egy noop output stream. Példa van rá az `EntityReader` osztályban. Az mehet nyugodtan egy `BeforeEach` metódusban, egy **nem** statikus mezőbe belebiggyesztve.